.github/workflows/prerelease-exit.yml

```diff
@@ -0,0 +1,37 @@
+name: Exit Prerelease Mode
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Exit prerelease mode on release branch"
+        type: string
+        default: "main"
+        required: true
+
+jobs:
+  exit_prerelease:
+    name: Changesets Exit Prerelease
+    runs-on: ubuntu-latest
+    # Allow GITHUB_TOKEN to have write permissions
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v3
+        name: Checkout Repo
+        with:
+          submodules: recursive
+
+      - name: "Setup"
+        uses: ./.github/actions/setup
+        
+      - name: Remove pre.json
+        run: pnpm changeset pre exit
```

Comment:
why remove?

Reply:
Removing the pre.json is how you exit prerelease mode into normal release mode

Reply:
@roninjin10 but in this case we want to keep the `-alpha` ?

Reply:
@roninjin10 want to make sure we're aligned here and then can merge. Anyway to test this? 🤔

Reply:
looking at this with fresh eyes I realize we likely could just not include the ability to exit prerelease mode as an automatic github action. We are only going to do it 1 time (when we exit alpha) or possibly 0 times (when it gets upstreamed to viem) so this can likely just not include the ability to publish in non prerelease mode tbh and then manually exit prerelease mode with a normal PR.

As for testing, unfortunately there really isn't a good way to test this (no dry runs available in changesets). This general setup has been tested in the EVMts monorepo and latticexyz/mud monorepo though so we can expect no issues here except the following flavors of issues:
- Copy pasta that we failed to change e.g. forgot to change the name of the repo to op-viem
- Failed to set secrets in github or  permissions of what the action can do is wrong

Reply:
Looking at it again I'm right we don't need this but :shrug: it's here and this repo is getting nuked eventually for viem or op so seems not worth thinking too hard about